README.md

```diff
@@ -0,0 +1,36 @@
+# A guide on how to use Try Enarx 🎯
+
+A little amount of efforts are required but we will walk you through all of it. For instance, trying enarx with your own WASM files is a matter of few clicks. Our goal is to understand how it 
+happens step by step by *learning and doing*.
+
+# Few insights on Try Enarx
+* Enarx supports [Webassembly](https://webassembly.org/) by using wasmtime as a runtime. It heavily works on it. So try to find out what these terms actually mean to us and why we use it?
+* Converting your files into WASM file can be easily done with the help of [Webassembly-guide](https://enarx.dev/docs/WebAssembly/Introduction). Also,with the comfort of your 
+  familiar language. 
+* Choose the language and implements Webassembly to it and then we're good to go( to try enarx with it).
+
+# Steps involed are: (also mentioned in a video)
+
+#### Step 1:  
+Go to [try.enarx.dev](https://try.enarx.dev/)
+
+#### Step 2:  Click on "Start Now" on the Try enarx page and then choose a platform.
+You can choose the following platforms: Intel/AMD on Equinix or Azure .
+
+#### Step 3:  
+Log in from your Github account.
+
+#### Step 4: 
+Choose a WASM file or the file you've converted into a WASM file.
+
+#### Step 5:
+ And Deploy🚀
+
+
+## You did it 👏
+
+Tadaa. Now your application is running on a safe ENARX Keep. Congrats on your first deploy🔥
+Feel free to join us over out [chat](https://chat.enarx.dev/channel/mentorship) and resource to our [Enarx](https://enarx.dev/docs/Start/Introduction) project will help you dive deep into our motive and perspective.
+
+
+
```

Comment:
Maybe remove the extra new lines at the end, leaving only one?

Reply:
For sure